.circleci/config.yml

```diff
@@ -15,8 +15,16 @@ jobs:
           name: List Data Directory Contents
           command: ./scripts/run_in_ci.sh ls data/testing
 
-    #### Add your analysis here ####
-    
+      - run:
+          name: Mutect2 vs Strelka2
+          command: ./scripts/run_in_ci.sh Rscript -e "rmarkdown::render('analyses/mutect2-vs-strelka2/01-set-up.Rmd', clean = TRUE)"
+
+      - run:
+          name: Mutect2 vs Strelka2 part 2
+          command: ./scripts/run_in_ci.sh Rscript -e "rmarkdown::render('analyses/mutect2-vs-strelka2/02-analyze-concordance.Rmd', clean = TRUE)"
```

Comment:
Can you add this above?

Reply:
Then you could delete this extra analysis. It'd be nice to have all of the runs for one analysis under the same header.

Reply:
Will do.